cla_backend/apps/reports/api.py

```diff
@@ -14,7 +13,6 @@ class ExportListView(generics.ListAPIView, generics.DestroyAPIView):
     model = Export
     permission_classes = (IsAdminUser,)
     authentication_classes = (SessionAuthentication,)
-    page_size = 1000
```

Comment:
Why wasn't `pagination_class` added to this viewset?

Reply:
We don't think that this one was being used in master - we added in pagination and then took it out again as adding in pagination actually resulted in a difference to master. We did a one off automated test adding in over 1000 rows to the table of exports and neither master or the new branch seemed to do any pagination.